solarforecastarbiter/reports/templates/html/body.html

```diff
@@ -107,17 +107,22 @@ <h3 id="observations-and-forecasts">Observations and Forecasts</h3>
 indicate smaller percentiles and darker colors indicate larger percentiles.
 Use the hover tool to determine the percentile.</p>
 {% endif %}
-<div id="scatter-div"></div>
 
 <script>
 ts_plot = JSON.parse('{{ timeseries_spec | safe }}');
 Object.assign(ts_plot,{config: {responsive: true}});
 Plotly.newPlot("timeseries-div", ts_plot);
+</script>
+{% endif %}
+
+<div id="scatter-div"></div>
```

Comment:
mention in the what's new that the ordering of timeseries and scatter plots has changed

Reply:
Order here hasn't changed, Just moved the scatter script to it's own conditional block since it's not possible to have a timeseries without a scatter plot, if the user only has access to the forecast values.